tests/flags/flags.go

```diff
@@ -65,6 +67,7 @@ func init() {
 	flag.StringVar(&PreviousReleaseTag, "previous-release-tag", "", "Set tag of the release to test updating from")
 	flag.StringVar(&PreviousReleaseRegistry, "previous-release-registry", "index.docker.io/kubevirt", "Set registry of the release to test updating from")
 	flag.StringVar(&ConfigFile, "config", "tests/default-config.json", "Path to a JSON formatted file from which the test suite will load its configuration. The path may be absolute or relative; relative paths start at the current working directory.")
+	flag.StringVar(&ArtifactsDir, "artifacts", os.Getenv("ARTIFACTS"), "Directory for storing reporter artifacts like junit files or logs")
```

Comment:
> Yes, I meant that they should not be referenced in the container bodied or definitions, only in the test running functions (BeforeEach/AfterEach/It).
> Maybe initializing a structure with getters instead of simple variable may do the trick.

I want to address that separately. We have this issue with all values which we influence via flags. I think it is enough to move all global variables to a common global config struct. A global `var TestConfig *TestConfig` should then be good enough. It would be set to the config at some point and anything which calls it before being initialized will just panic due to a nil exception.

Reply:
Added to the commit message.